README.md

```diff
@@ -37,8 +37,7 @@ The `oColorsGetColorFor` function takes three arguments:
 
 * **Use case list**: a list of colour use cases in order of preference.  The first one that is defined for the specified property will be returned
 * **Property**: The property that you want to use the colour for (background, border, or text).  Note that in contrast to the `oColorsFor` mixin, you must specify only one property.   Options are `background`, `border`, `text`, and `all`.
-* **Options**: A Sass *map* of additional options, all of which are optional, and may comprise:
-	* **default**: The name of a palette colour to return if none of the specified use cases are defined for the desired property.  May also be set to `null` or `undefined` to return that instead of the built in default (which is transparent)
+* **Default**: The name of a palette colour to return if none of the specified use cases are defined for the desired property.
```

Comment:
I would suggest leaving it as it was.  It's an extensible options map, so it doesn't get in the way of future things, is currently used internally, works, no need to change.